nasa-gibs/worldview
# WV-2849: Add OPERA vegetation disturbance layer

## Description

Fixes #WV-2849 .

Add OPERA Vegetation Disturbance Alert layer. 

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-72.34390941141912,-35.23542556551287,-71.92244219729521,-34.98867873855248&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OPERA_L3_DIST-ALERT-HLS_Color_Index(disabled=9-0),HLS_L30_Nadir_BRDF_Adjusted_Reflectance,HLS_S30_Nadir_BRDF_Adjusted_Reflectance,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2023-01-01-T16%3A09%3A42Z)
2. Check to see the imagery displays for 2023-JAN-01 to 2023-JAN-03 - both the HLS Land Surface Reflectance and the OPERA Vegetation Disturbance Status. See that the values change as you progress from Jan 1 to Jan 3 (showing confirmed vegetation loss)
3. Also check the layer description, Data download works, image snapshot works (even though it will ask to revert to the original layer configuration *this is scheduled to be fixed in sprint 5)

@nasa-gibs/worldview
